app/containers/message/Components/Audio/Slider.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { LayoutChangeEvent, View, TextInput } from 'react-native';
+import { PanGestureHandler } from 'react-native-gesture-handler';
+import Animated, {
+	SharedValue,
+	runOnJS,
+	useAnimatedGestureHandler,
+	useAnimatedProps,
+	useAnimatedStyle,
+	useDerivedValue,
+	useSharedValue
+} from 'react-native-reanimated';
+
+import styles from './styles';
+import { useTheme } from '../../../../theme';
+
+const AnimatedTextInput = Animated.createAnimatedComponent(TextInput);
+
+interface ISlider {
+	duration: SharedValue<number>;
+	currentTime: SharedValue<number>;
+	loaded: boolean;
+	onChangeTime: (time: number) => Promise<void>;
+}
+
+const BUTTON_HIT_SLOP = {
+	top: 8,
+	right: 8,
+	bottom: 8,
+	left: 8
+};
+
+const Slider = ({ currentTime, duration, loaded = false, onChangeTime }: ISlider) => {
```

Comment:
Can you rename this to `Seek`? It's the standard name for this.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio#events